### PR TITLE
Issue SB-30516 fix: Added code to get video duration

### PR DESCRIPTION
**Pull request by Xitija**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Xitija** [comment]
@HarishGangula 
Sir I am sending this change for getting duration of the video.

cc: @vinukumar-vs @itsvick

File: projects/sunbird-video-player/src/lib/components/video-player/video-player.component.ts [hunk 1]
```diff
@@ -230,8 +230,11 @@ export class VideoPlayerComponent implements AfterViewInit, OnInit, OnDestroy, O
     });
     events.forEach(event => {
       this.player.on(event, (data) => {
+        if (this.totalDuration === 0 && data.type === 'durationchange') {
```
Comment (vinukumar-vs) [inline projects/sunbird-video-player/src/lib/components/video-player/video-player.component.ts:233]: This is the generic method to dispatch all the events. This logic should not be present here. 

As discussed, of you need only duration verify the root where we are sending the duration & fix that code if you have any issue.

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

**Comment by HarishGangula** [comment]
@Xitija  Test cases are missing for this PR
cc: @vinukumar-vs

**Comment by Xitija** [comment]
> @Xitija Test cases are missing for this PR cc: @vinukumar-vs

Adding test cases

**Pull request merged**